modules/genomic_browser/js/FileUploadModal.js

```diff
@@ -10,19 +12,19 @@ var GenomicFileUploadModal = React.createClass({
     baseURL: React.PropTypes.string.isRequired
   },
 
-  getInitialState: function () {
+  getInitialState: function getInitialState() {
```

Comment:
I think we already raised that question, and I found a decent explanation for such behaviour here: http://stackoverflow.com/a/35857404